protos/sources/vega/events/v1/events.proto

```diff
@@ -13,6 +13,16 @@ import "vega/vega.proto";
 
 option go_package = "code.vegaprotocol.io/vega/protos/vega/events/v1";
 
+message PartyActivityStreak {
```

Comment:
This might require some doc comments

Reply:
hum that's an event, we often leave events undocumented.

Reply:
addressed